.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build & Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install system dependencies
+      run: npm install -g @webosose/ares-cli
```

Comment:
This is very likely not required - we already have a pinned dependency on `@webosose/ares-cli` in our `package.json`, and it should be accessible in `npm` scripts used below (or directly via `node_modules/.bin/ares-...`)

Reply:
Ah I see! Correcting that.